daemons/lvmdbusd/manager.py

```diff
@@ -139,11 +139,20 @@ def FlightRecorderDump(self):
 		Dump the flight recorder to syslog
 		"""
 		cfg.blackbox.dump()
+
+	@staticmethod
+	def _lookup_by_lvm_id(key):
+		p = cfg.om.get_object_path_by_uuid_lvm_id(key, key)
+		if p:
+			return p
+		return '/'
```

Comment:
What about logging this as ``debug`` too? That's what I did to debug the issues.